changelog/6991.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix --lf with explicit cmdline args.
```

Comment:
```suggestion
Fix `--lf` for files explicitly given in the command-line.
```

Reply:
s/in/on ?

Anyway, I will add the other changes here also, and reword that then.